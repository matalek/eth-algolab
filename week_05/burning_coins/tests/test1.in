10
1
1
10
9 434 68 987 527 853 134 849 301 723
10
8 391 158 362 23 829 892 710 408 399
10
7 695 188 389 828 19 243 268 161 815
10
6 644 147 951 646 818 745 148 370 629
10
5 890 695 998 451 416 998 343 424 290
10
4 178 169 717 836 391 506 644 105 874
10
3 331 113 257 541 844 463 300 707 257
10
2 669 60 134 997 770 567 674 149 608
10
1 736 997 352 712 520 331 980 551 73
