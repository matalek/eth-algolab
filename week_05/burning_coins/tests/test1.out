1
3846
2691
2186
3190
2937
2804
1989
2855
2664
